### PR TITLE
Feature/21 seed url data

### PR DESCRIPTION
**Pull request by bryanakitchen**
Adds initial Links/URL data to tables with initial data, create tables, drop tables, and load seed.

### CODE REVIEW

File: data/tips-data.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    tip: 'Pause from your work and take a few deep breaths. Inhale through your nose for five seconds. Exhale through your mouth for five seconds. Repeat five times.'
+  },
+  {
+    tip: 'Give your eyes a break. Find an object that is about twenty feet away and focus on it for twenty seconds.'
+  },
+  {
+    tip: 'Let\'s take a break. Step away from your computer (and any other screens) for 5-10 minutes.'
+  },
+  {
+    tip: 'Time for a break. Do nothing for the next two minutes. (See http://www.donothingfor2minutes.com/ for guidance) '
```
Comment (ShaneUP1) [inline data/tips-data.js]: I think I'd rather leave out links on the quick tips to limit their distractions and clicks. We can just borrow any of the tips from that site and insert instead.

**Review by ShaneUP1 — APPROVED** [review]
Left 1 comment, but looks great! THanks!

File: data/drop-tables.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+run();
+
+async function run() {
+
+  try {
+    await pool.connect();
+
+    await pool.query(`
+        DROP TABLE IF EXISTS links CASCADE;
```
Comment (franco-ortega) [inline data/drop-tables.js]: Same question here as Feature/20. Is CASCADE needed if the tables are not joined?

**Review by franco-ortega — APPROVED** [review]

File: data/tips-data.js (hunk 1 above)
Comment (bryanakitchen) [reply data/tips-data.js]: I've updated this. I believe it could be helpful for focus/limiting distraction.

**Pull request merged**